### PR TITLE
Another couple of postgis-related changes in support of a staging env

### PR DESCRIPTION
**Pull request by matschaffer**
The db.rake part of this might not be necessary. Trying a new build of of staging now with an updated manifest from c66 support.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Send test-coverage to code climate
 require "codeclimate-test-reporter"
```
Comment (evadne) [inline spec/spec_helper.rb]: Perhaps also invoke `require` only when there is a token

File: lib/tasks/db.rake [hunk 2]
```diff
@@ -2,8 +2,19 @@ namespace :db do
   #NOTE: Newer versions of rails should support this in schema.rb. Ours doesn't atm.
   desc 'Creates the postgis extension on the datatabase'
   task :enable_postgis => :environment do
-    connection = ActiveRecord::Base.connection
-    connection.exec_query('CREATE EXTENSION IF NOT EXISTS postgis')
+    # for some reason `create extension if exists` doesn't work consistently
```
Comment (evadne) [inline lib/tasks/db.rake]: IMO this is unnecessary if we go with `structure.sql`

Comment (matschaffer) [inline lib/tasks/db.rake]: It is, that's why I closed the PR ;)

File: spec/spec_helper.rb (hunk 1 above)
Comment (matschaffer) [inline spec/spec_helper.rb]: Good call. Opened as #215

**Pull request closed without merging**